modules/DMZ/data_kit/filler_regression.py

```diff
@@ -1,15 +1,24 @@
 from modules.DMZ.data_kit import data_splitting
 from modules.DMZ.data_kit import data_filler
+from sklearn import svm
+import xgboost
+import pandas as pd
 
 
 class RegressionFiller(object):
 
     def __init__(self, train, test):
-        self.complete_data = self.select_complete_data(train)
-        self.incomplete_data = self.select_incomplete_data(train)
-        self.testing_data = test
-        self.select_missing_column(self.incomplete_data)
-        self.select_missing_column(self.complete_data)
+        self.model = None
+        self.filled_data = None
+        self.filled_test_data = None
+        self.train_data = train
+        self.test_data = test
+
+        self.total_data = pd.concat([train, test], ignore_index=True)
+        self.total_missing = self.select_incomplete_data(self.total_data)
+        self.total_data = self.select_complete_data(self.total_data)
+        self.filled_data = self.fill_missing_columns(self.train_data, self.total_data)
```

Comment:
So the `fill_missing_columns` gets called twice? Is it consistent? Or will there be a chance that the method would fill in the missing data each time its run?